src/lib/integration_test_cloud_engine/kubernetes_network.ml

```diff
@@ -245,8 +256,14 @@ module Node = struct
       | Some peer ->
           Malleable_error.return peer#peerId
     in
-    let peers_ids = (query_result_obj#daemonStatus)#peers in
-    return (self_id, Array.to_list peers_ids)
+    (* [%log info] "get_peer_id, self_id is: %s" self_id ; *)
```

Comment:
Remove these comments please.

Reply:
those are just log lines, i figured i'd just comment them out and leave it there so that if/when the next time we need to debug we can just comment it back in

Reply:
We tend to avoid leaving code commented in our code, so I would lean towards removal. Though I understand your want to have these around for the future. Instead of removing them, you could add them all back in so long as they are all downgraded to `debug` instead of `info`. Then they are easy to filter out by default using logproc.